### ISSUE TITLE
Is Twidere Project Dead?

### ISSUE THREAD

**Issue by alphanerdgen** [TwidereProject/TwidereX-Android#450]
It's been a long time since both the apps received an update so if this project is dead then do let us know.

**Comment by cunninglemming** [comment]
[1.5.1](https://github.com/TwidereProject/TwidereX-Android/releases/tag/1.5.1) 751 commits to develop since this release

Not dead but it's been a while since last release. Any chance we'll get a new one in near future?

**Comment by Tlaster** [comment]
No, we're just short of developers since we have multiple projects to do but we only have limited members. Twidere X is still a thing in our plan, sorry for the delay.
And we're still hiring more and more developers so that we can maintain a good upgrade schedule for these projects.

**Comment by alphanerdgen** [comment]
Yeah, thought as much. Anyway, glad the project is still alive and kicking. Thanks a lot for dropping the latest update and, of course, for all your hard work. Cheers!
